Exercise1_Test.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import Exercise1
+import unittest
+
+class Exercise1Test(unittest.TestCase):
+
+  def testBaseCases(self):
+    self.assertEquals(True, Exercise1.is_permutation("", ""))
+    self.assertEquals(False, Exercise1.is_permutation(None, None))
+
+  def testKnownCases(self):
```

Comment:
I have added a few more tests for the method that handles only English and some more tests for the base case. I also wanted to test how the maximum int value will be dealt with, but my test never terminates. See below:
`def testErrorCases(self):
    value = ""
    for counter in range(0, sys.maxsize) :
        value += "a"
    self.assertTrue(Exercise1.is_permutation(value, value))
`